examples/with-ecs-scheduling/README.md

```diff
@@ -0,0 +1,65 @@
+# EventBridge ECS & Scheduled Events Example
+
+Configuration in this directory creates EventBridge resource configuration including an ECS service.
+
+This requires you to have setup an ECS cluster and VPC and passed them in via variables.
```

Comment:
The examples are always self-containing and require zero-configuration - VPC/ECS and other minimally required resources should be created by this example (see other complex modules in terraform-aws-modules, eg RDS Aurora, Lambda).

Reply:
If VPC is not important, you can use data source `aws_vpc` to fetch the default VPC (see `security-group` module examples).